sub/ass_mp.c

```diff
@@ -134,16 +138,12 @@ static void message_callback(int level, const char *format, va_list va, void *ct
 
 ASS_Library *mp_ass_init(struct mpv_global *global, struct mp_log *log)
 {
-    char *path = mp_find_config_file(NULL, global, "fonts");
     mp_dbg(log, "ASS library version: 0x%x (runtime 0x%x)\n",
            (unsigned)LIBASS_VERSION, ass_library_version());
     ASS_Library *priv = ass_library_init();
     if (!priv)
         abort();
     ass_set_message_cb(priv, message_callback, log);
-    if (path)
-        ass_set_fonts_dir(priv, path);
-    talloc_free(path);
```

Comment:
This should probably stay here in `mp_ass_init`. You can use `mp_get_config_group` to fetch a copy of the opts you need and go from there.

Reply:
If I'm not mistaken, the opts will have to be passed as arguments, since `mp_ass_init` is called for both subs and OSD, each having their own opts.  (Please correct me if I'm wrong; `mp_get_config_group` is somewhat of a black box to me, and I can't figure out if it could do the job with their respective `mpv_global` alone.)

(Still, even with the additional argument, the end result is much simpler, so that's a net win.)

Reply:
Yep your way is much better. I thought there was only one `mp_ass_init` call.